.gitignore

```diff
@@ -475,3 +475,6 @@
 
 # /utils/wxrc/
 /utils/wxrc/*.sln
+
+# JetBrain's CLion
+.idea
```

Comment:
We don't put generic exclusions like this in this file, it doesn't make sense to have them in every `.gitignore` of each project -- please put it in your global `.gitignore` (i.e. the one pointed to by `core.excludesfile`) instead.